src/components/RightDockedModal.js

```diff
@@ -15,25 +15,25 @@ const propTypes = {
     children: PropTypes.node.isRequired,
 
     // Route constant to show modal
-    route: PropTypes.string,
+    routes: PropTypes.arrayOf(PropTypes.string),
 
     /* Onyx Props */
     // Url currently in view
     currentURL: PropTypes.string,
 };
 
 const defaultProps = {
-    route: '',
+    routes: [],
     currentURL: '',
 };
 
 const RightDockedModal = memo(({
-    route, children, currentURL,
+    routes, children, currentURL,
 }) => (
     <Modal
         type={CONST.MODAL.MODAL_TYPE.RIGHT_DOCKED}
         onClose={redirectToLastReport}
-        isVisible={currentURL === route}
+        isVisible={routes.includes(currentURL)}
```

Comment:
Hmm this is definitely going to create issues since there is a conflicting implementation here:

https://github.com/Expensify/Expensify.cash/pull/1630/files

We are adding fuzzy matching for the routes there. This PR will undo that.

Why do we need to provide multiple routes here?

Reply:
Ah, thanks for the heads up. Removing the route elsewhere makes this irrelevant, so I rolled back this change.